app/models/user.rb

```diff
@@ -369,13 +369,12 @@ def cached_followed_tag_names
     end
   end
 
-  # methods for Administrate field
-  def banned
-    has_role? :banned
+  def suspended?
+    has_role?(:suspended) || has_role?(:banned)
```

Comment:
should we put a TODO here are well to clean up the banned check once this is merged and the data update script has run?